derrida/books/tests/test_views.py

```diff
@@ -1008,32 +985,6 @@ def test_get_proxy_url(self, mockrequests):
         assert str(imgurl) == str(cover.image \
                 .region(x=0, y=0, width=2048, height=2048).size(width=1024))
 
-        # large image restricted to certain images only
```

Comment:
did we lose any logic / testing about the large images being restricted? (I don't see it in the following code, but would like confirmation!)